tests/filesTest.js

```diff
@@ -1413,29 +1500,19 @@ describe('Main: Currently testing files management,', function () {
             ocFilterFiles.appendElement('d:prop', '', dProp => {
               dProp.appendElement('oc:fileid', '', '')
             }).appendElement('oc:filter-rules', '', ocFilterRules => {
-              ocFilterRules.appendElement('oc:systemtag', '', tagId)
+              ocFilterRules.appendElement('oc:systemtag', '',
+                MatchersV3.fromProviderState('${tagId}', tagId)) /* eslint-disable-line no-template-curly-in-string */
             })
           })
         })
         .willRespondWith(getFileInfoBy('tag'))
 
-      await getCapabilitiesInteraction(provider)
-      await getCurrentUserInformationInteraction(provider)
-
       return provider.executeTest(async () => {
-        const oc = createOwncloud()
+        const oc = createOwncloud(config.testUser, config.testUserPassword)
         await oc.login()
 
-        return oc.files.fileInfo(newFile, ['{http://owncloud.org/ns}fileid'])
-          .then(fileInfo => {
-            fileId = fileInfo.getFileId()
-            return oc.systemTags.createTag({ name: newTagName })
-          }).then(resp => {
-            tagId = resp
-            return oc.systemTags.tagFile(fileId, tagId)
-          }).then(() => {
-            return oc.files.getFilesByTags([tagId], ['{http://owncloud.org/ns}fileid'])
-          }).then(files => {
```

Comment:
tests are fileInfo and getFileID are already present, but I dont see any test for createTags, You need to add that. Or you can leave that test as it is and refactor this later

Reply:
done in PR https://github.com/owncloud/owncloud-sdk/pull/792
waiting that to be merged